openpype/hosts/maya/plugins/publish/extract_camera_mayaScene.py

```diff
@@ -172,18 +172,19 @@ def process(self, instance):
                                                       dag=True,
                                                       shapes=True,
                                                       long=True)
+
+                    attrs = {"backgroundColorR": 0.0,
+                             "backgroundColorG": 0.0,
+                             "backgroundColorB": 0.0,
+                             "overscan": 1.0}
+
                     # Fix PLN-178: Don't allow background color to be non-black
-                    for cam in cmds.ls(
+                    for cam, (attr, value) in itertools.product(cmds.ls(
                             baked_camera_shapes, type="camera", dag=True,
-                            shapes=True, long=True):
-                        attrs = {"backgroundColorR": 0.0,
-                                 "backgroundColorG": 0.0,
-                                 "backgroundColorB": 0.0,
-                                 "overscan": 1.0}
-                        for attr, value in attrs.items():
-                            plug = "{0}.{1}".format(cam, attr)
-                            unlock(plug)
-                            cmds.setAttr(plug, value)
+                            long=True), attrs.items()):
+                        plug = "{0}.{1}".format(cam, attr)
+                        unlock(plug)
+                        cmds.setAttr(plug, value)
```

Comment:
I know it's functionally the same - but do we feel `itertools.product` is really that much more readable then the nested for loop? I'm fine with either but somehow I have a harder time reading this new code.